src/main/java/net/dv8tion/jda/internal/entities/PrivateChannelImpl.java

```diff
@@ -82,6 +82,12 @@ public long getLatestMessageIdLong()
         return latestMessageId;
     }
 
+    @Override
+    public boolean canTalk()
+    {
+        return !getUser().isBot() || getJDA().getAccountType() != AccountType.BOT;
```

Comment:
Bots can talk in dms?

Reply:
Changed it with true since it's possible